docs/panel/advanced/docker.mdx

```diff
@@ -0,0 +1,189 @@
+
+This configuration includes an integrated web server that will automatically obtain SSL certificates if you are serving over HTTPS. For the database, it assumes you want to use SQLite (or you have an external database server to configure using the  installer.) It also assumes you intend to use the Filesystem driver for cache, filesystem or database driver for session, and database driver for queue (or you have an external Redis server to configure using the installer.) If you want to use other options built into Docker, see [Advanced Options](#advanced-options).
+
+### Create compose.yml
+
+```yml title="compose.yml"
+services:
+  panel:
+    image: ghcr.io/pelican-dev/panel:latest
+    restart: always
+    networks:
+      - default
+    ports:
+      - "80:80"
+      - "443:443"
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+    volumes:
+      - pelican-data:/pelican-data
+      - pelican-logs:/var/www/html/storage/logs
+    environment:
+      XDG_DATA_HOME: /pelican-data
+      APP_URL: "http://localhost"
+      ADMIN_EMAIL: "USEYOUROWNEMAILHERE@example.com"
+
+volumes:
+  pelican-data:
+  pelican-logs:
+
+networks:
+  default:
+    ipam:
+      config:
+        - subnet: 172.20.0.0/16
+```
+
+### Set Required Environment Variables
+1. Set `APP_URL` to the base URL your server will be reachable on, including the protocol (https:// or http://) and port. 
+    - Note that Caddy, the integrated webserver, will serve a 308 redirect to any requests on port 80 if the `APP_URL` begins with `https://`. If your final site will be reachable over HTTPS but TLS (SSL) will be handled and terminated by an upstream server, such as a reverse proxy, you will need to use a [custom caddyfile](#custom-caddyfile).
+2. Set the `ADMIN_EMAIL` to your email address. Caddy will use this email address to generate a LetsEncrypt SSL certificate if you are serving via HTTPS.
+
+Now, close and save changes to `compose.yml`. 
+
+### Starting
+From the directory in which the Caddyfile is located, run:
+
+```sh
+docker compose up -d
+```
+
+### Back Up Your Encryption Key
+The first time the container starts, it will generate an `APP_KEY` which is used as an encryption key. This will be saved automatically, but you should save a copy in a secure place in case you need it later.
+
+Run:
+
+```sh
+docker compose logs panel | grep 'Generated app key:'
+```
+
+### Installing
+Open the installer in your browser at `APP_URL/installer` to finish setting up the panel.
+
+:::note
+The first time the container starts after installing or updating, it will apply database migrations, which may take a few minutes. The panel will not be accessible during this process.
+:::
+
+#### Sensible Driver Defaults:
+
+* Cache Driver: Filesystem
+* Database Driver: SQLite
+* Queue Driver: Database
+* Session Driver: Filesystem
+
+For other configuration, such as UI options, CAPTCHA, email, backups and OAuth, head to the settings menu in the admin panel.
```

Comment:
```suggestion
```

Reply:
Can I ask why on this one? This is definitely information I would be looking for when following similar guides.